managed/ui/src/components/config/PublicCloud/ProviderConfiguration.js

```diff
@@ -331,7 +332,10 @@ class ProviderConfiguration extends Component {
     } else if (this.state.currentView === 'result') {
       currentProviderView = this.getResultView();
     }
-    return <div className="provider-config-container">{currentProviderView}</div>;
+    return <>
+      <div className="provider-config-container">{currentProviderView}</div>
+      <ToastContainer />
```

Comment:
there should be a single `ToastContainer` component in the application, preferably somewhere in `App.js`

Reply:
Done